.github/workflows/semver-checks-pr-label.yml

```diff
@@ -0,0 +1,81 @@
+on:   # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["Check semver breaks"]
+    types:
+      - completed
+
+name: Check semver breaks - Label and Comment PR
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'failure'
+    steps:
+      - name: "Download artifact"
+        uses: actions/github-script@v7
+        with:
+          script: |
+            let fs = require('fs');
+            let allArtifacts = await github.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
+            let matchArtifacts = allArtifacts.data.artifacts.filter((artifact) => {
+              return artifact.name.startsWith('pr');
+            });
+            for (let artifact of matchArtifacts) {
+              console.log(artifact.name);
+              let download = await github.actions.downloadArtifact({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                artifact_id: matchArtifact.id,
+                archive_format: 'zip',
+              });
+              fs.writeFileSync('${{github.workspace}}/' + artifact.name + '.zip', Buffer.from(download.data));
+            }
+      - name: "Unzip artifact(s)"
+        run: unzip pr*.zip
+
+      - name: "Comment and add label on PR - Semver break"
+        uses: actions/github-script@v7
+        if: ${{ hashFiles('semver-break-NR') != '' }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            var fs = require('fs');
+            var issue_number = Number(fs.readFileSync('./semver-break-NR'));
+            await github.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              body: ':rotating_light: API BREAKING CHANGE DETECTED'
+            });
+            await github.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              labels: ['API break']
+            });
```

Comment:
Is it possible to avoid JS please? :)

Reply:
The examples you linked are all in JS. To be honest even with this little exposure to JS I might need to swallow some potassium iodine over the weekend.

Reply:
Here's some code from my project that works with artifact without JS: https://github.com/debian-cryptoanarchy/cryptoanarchy-deb-repo-builder/blob/master/.github/workflows/ci.yml

I think you can simply store `break` or `pass` in a file, upload it in one job then download it in the other and use `if [ "`cat result`" = "break" ]` to check the result.

Reply:
Yes, but then how do I create labels and add comments?
Hacking the `.github/labeler.yml` seems overkill.

Reply:
Use labels in your own repo?